### PR TITLE
add my theme:idk

### PR DESCRIPTION
**Pull request by UniFreak**

### CODE REVIEW

File: _posts/2015-09-25-idk.markdown [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+layout: post
+title: idk
+date: 2015-09-25 14:35:00
+homepage: https://github.com/UniFreak/unifreak.github.io
+download: https://codeload.github.com/UniFreak/unifreak.github.io/zip/master
+demo: https://unifreak.github.io
+author: FangHao
+thumbnail: idk.png
+license: MIT
```
Comment (gynter) [inline _posts/2015-09-25-idk.markdown]: Please change the license value to `MIT License` so similar formatting is used for all of themes. Thanks!

**Comment by gynter** [comment]
The image size is currently 249px × 200px, please change it to 250px × 200px.

**Comment by UniFreak** [comment]
done :)

**Comment by gynter** [comment]
Thanks! @mattvh :shipit:

**Pull request merged**